src/Http/Error.php

```diff
@@ -55,6 +55,11 @@ public function __construct(
         $lorisInstance = $request->getAttribute('loris');
         $user          = $request->getAttribute('user') ?? new \LORIS\AnonymousUser();
 
+        // Variables used to suggest the user to login and later redirect them if they
+        // are not authenticated in a 403.
+        $tpl_data['anonymous'] = $user instanceof \LORIS\AnonymousUser;
+        $tpl_data['url'] = urlencode($_SERVER['REQUEST_URI']);
```

Comment:
Also, I used `$_SERVER['REQUEST_URI']` instead of the `$uri` object here because the latter seems to return a `lorispath=url` instead of the URL I want directly. I haven't been able to find why that is used so if anyone has an explanation I am all ears.

Reply:
That sounds like the `mod_rewrite` rule from Apache leaking through somewhere. That should not happen. How were you testing it / in what kind of environment that you had that problem?

Reply:
> That sounds like the `mod_rewrite` rule from Apache leaking through somewhere. That should not happen. How were you testing it / in what kind of environment that you had that problem?

I am using a MCIN VM with Apache 2.4 and PHP 8.1. I did not think there was anything wrong with my installation so this is certainly puzzling.